src/platforms/rainier/flett.cpp

```diff
@@ -44,8 +44,8 @@ void FlettNVMeDrive::unplug([[maybe_unused]] Notifier& notifier, int mode)
         removeFromInventory(inventory);
     }
     drive.reset();
-    debug("Drive {NVME_ID} unplugged on Williwakas {WILLIWAKAS_ID}", "NVME_ID",
-          index, "WILLIWAKAS_ID", flett->getIndex());
+    debug("Drive {NVME_ID} unplugged on Flett {FLETT_ID}", "NVME_ID", index,
```

Comment:
it does sound a little funny since physically the drivers plug into williwakas

Reply:
Yeah I know, but it's more confusing when trying to read the logs and figuring out what went wrong :)